src/sentry/static/sentry/app/views/settings/components/forms/choiceMapperField.jsx

```diff
@@ -230,10 +224,46 @@ export default class ChoiceMapper extends React.Component {
   }
 }
 
-const LabelColumn = styled(p => <Box flex="0 0 200px" {...p} />)``;
+const Header = styled('div')`
+  display: flex;
+  align-items: center;
+`;
+
+const Heading = styled('div')`
+  display: flex;
+  margin-left: ${space(1)};
```

Comment:
Please, are you sure about this margin? `space(1)` returns` 8px` and before this margin was` 1px`

Reply:
Reflex box's `m*` and `p*` attributes used a similar spacing config

Reply:
Yeah I didn't know... Billy told me the same yesterday... thanks :)